lib/split/dashboard.rb

```diff
@@ -32,6 +33,11 @@ class Dashboard < Sinatra::Base
       erb :index
     end
 
+    post '/initialize_experiment' do
+      Split::ExperimentCatalog.find_or_create(params[:experiment]) unless params[:experiment].nil? || params[:experiment].empty?
```

Comment:
Super tiny nitpick - would the following work? I haven't actually tested it myself, so it might not be 100% equivalent, but throwing it out there...

```suggestion
      Split::ExperimentCatalog.find_or_create(params[:experiment]) if params[:experiment].present?
```

Reply:
So interesting thing about this is that present? is that it is in active_support which is not in the Split gem, so when I used present? originally it would throw an error. Split uses Sinatra not Rails unfortunately.